extension/manifest.json

```diff
@@ -67,32 +66,41 @@
     }
   ],
   "background": {
-    "page": "/js/background_page/background_page.htm"
+    "service_worker": "/js/service_worker/background.js",
+    "type": "module"
   },
   "options_ui": {
     "page": "/chrome/settings/index.htm",
     "open_in_tab": true
   },
   "web_accessible_resources": [
-    "/css/webmail.css",
-    "/img/svgs/reply-icon.svg",
-    "/img/svgs/spinner-white-small.svg",
-    "/img/svgs/spinner-green-small.svg",
-    "/img/svgs/unlock.svg",
-    "/img/logo/flowcrypt-logo-19-19.png",
-    "/chrome/elements/compose.htm",
-    "/chrome/elements/pgp_block.htm",
-    "/chrome/elements/setup_dialog.htm",
-    "/chrome/elements/attachment.htm",
-    "/chrome/elements/attachment_preview.htm",
-    "/chrome/elements/passphrase.htm",
-    "/chrome/elements/add_pubkey.htm",
-    "/chrome/elements/pgp_pubkey.htm",
-    "/chrome/elements/backup.htm",
-    "/chrome/elements/verification.htm",
-    "/chrome/elements/shared/footer.htm",
-    "/chrome/elements/oauth2.htm"
+    {
+      "resources": [
+        "/css/webmail.css",
+        "/img/svgs/reply-icon.svg",
+        "/img/svgs/spinner-white-small.svg",
+        "/img/svgs/spinner-green-small.svg",
+        "/img/svgs/unlock.svg",
+        "/img/logo/flowcrypt-logo-19-19.png",
+        "/chrome/elements/compose.htm",
+        "/chrome/elements/pgp_block.htm",
+        "/chrome/elements/setup_dialog.htm",
+        "/chrome/elements/attachment.htm",
+        "/chrome/elements/attachment_preview.htm",
+        "/chrome/elements/passphrase.htm",
+        "/chrome/elements/add_pubkey.htm",
+        "/chrome/elements/pgp_pubkey.htm",
+        "/chrome/elements/backup.htm",
+        "/chrome/elements/verification.htm",
+        "/chrome/elements/shared/footer.htm",
+        "/chrome/elements/oauth2.htm",
+        "/js/common/core/feature-config-injector.js"
+      ],
+      "matches": ["*://*/*"]
```

Comment:
The following "matches" directive with overlypermissive wildcards is something we do not want have access to the resources:

```
"matches": ["*://*/*"]
```

I have reviewed the links below regarding web_accessible_resources:

https://developer.chrome.com/docs/extensions/reference/manifest/web-accessible-resources
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/web_accessible_resources

And the best options for us is to replace that with:

```
"extension_ids": []
```

This will specific that only our extension can frame the FlowCrypt extension and no websites/extensions can frame them.

Reply:
Sounds good!